cpp/farm_ng/core/logging/logger.cpp

```diff
@@ -58,4 +54,17 @@ void StreamLogger::writeHeader(
 void StreamLogger::write(std::string const& str) { std::cerr << str; }
 void StreamLogger::flush() { std::cerr << std::endl; }
 
+const std::string StreamLogger::kDefaultHeaderFormat =
+    "[FARM {text} in {file}:{line}]\n";
+
+const StreamLogger::LogClock StreamLogger::kDefaultLogClock =
+    StreamLogger::LogClock{.now = []() {
+      auto now = std::chrono::system_clock::now();
```

Comment:
Since we're defaulting to system_time, shouldn't we also update the vns_app to initialize this with the `MissionClock::now()` call?

Reply:
This is in reference to a downstream project; please see its respective repository for that PR